apps/condo/domains/marketplace/utils/clientSchema/Invoice.ts

```diff
@@ -259,6 +258,9 @@ export function processRowsFromInvoiceTicketForm (rawRows, intl) {
         }
 
         const resultRow = { ...row, toPay: resultToPay }
+        if (!row.measure) {
+            resultRow.measure = undefined
+        }
```

Comment:
You will set `resultRow.measure` to `undefined` even if `row.measure` is 0, false, null, empty string. Is it OK?

Reply:
Anyway, strange 3-line logic :)